config.json

```
@@ -5,7 +5,8 @@
     "days_to_reschedule": 7,
     "auto_reschedule_after_sync": false,
     "auto_disperse_after_sync": false,
-    "auto_disperse": false,
+    "auto_disperse_when_review": false,
+    "auto_disperse_after_reschedule": true,
```

It breaks load balance. So, it shouldn't be enabled by default.

```suggestion
    "auto_disperse_after_reschedule": false,
```